aws-import/awsimport.py

```diff
@@ -0,0 +1,257 @@
+    ],
+    "elasticache": [
+        DocSet(TABLE,
+               # Make this a list so that the order is deterministic. There are some duplicate events across
+               # ElasticCache Redis and memcache so we just pick one.
+               ["https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/CacheMetrics.Redis.html",
+                "https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/CacheMetrics.HostLevel.html",
+                "https://docs.aws.amazon.com/AmazonElastiCache/latest/mem-ug/CacheMetrics.Memcached.html",
+                "https://docs.aws.amazon.com/AmazonElastiCache/latest/mem-ug/CacheMetrics.HostLevel.html"
+                ]),
+    ],
+    "elb": [
+        DocSet(TABLE, {"https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-cloudwatch-metrics.html"})
+    ],
+    "kinesis": [
+        DocSet(TABLE, {"https://docs.aws.amazon.com/streams/latest/dev/monitoring-with-cloudwatch.html"})
+    ],
+    "lambda": [
+        DocSet(TABLE, {"https://docs.aws.amazon.com/lambda/latest/dg/monitoring-functions-metrics.html"})
+    ],
+    "opsworks": [
+        DocSet(TABLE, {"https://docs.aws.amazon.com/opsworks/latest/userguide/monitoring-cloudwatch.html"})
+    ],
+    "rds": [
+        DocSet(TABLE, {"https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MonitoringOverview.html"})
+    ],
+    "redshift": [
+        DocSet(TABLE, {"https://docs.aws.amazon.com/redshift/latest/mgmt/metrics-listing.html"})
+    ],
+    "route53": [
+        DocSet(LIST, {"https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/monitoring-cloudwatch.html",
+                      "https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/monitoring-resolver-with-cloudwatch.html"}),
+    ],
+    "sns": [
+        DocSet(TABLE, {"https://docs.aws.amazon.com/sns/latest/dg/sns-monitoring-using-cloudwatch.html"})
+    ],
+    "sqs": [
+        DocSet(TABLE, {
+            "https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-available-cloudwatch-metrics.html"})
+    ],
+}
+
+
+class Fetcher:
+    def __init__(self):
+        self._sess = requests.Session()
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.close()
+
+    def fetch(self, url: str, extractor: Extractor) -> List[Metric]:
+        data = self._sess.get(url).content
+        tree = BeautifulSoup(data, "html.parser")
+        return list(extractor.extract(tree))
+
+    def close(self):
+        self._sess.close()
+
+
+def sync(name, metrics):
+    """Write markdown docs"""
+    for metric in metrics:
+        mdFile = f"{metric.name}.md"
+        docs = Path().resolve().parent / f"aws-{name}" / "docs"
+        docs.mkdir(exist_ok=True)
+
+        (docs / mdFile).write_text(dedent(f"""
+            ---
+            title: {metric.name}
+            brief: {metric.brief}
```

Comment:
After talking to Joe about it, I'm making the `brief` field the first sentence of the long description and then putting the long description in the body of the md below.

Reply:
The nltk library was a nice function that parses sentences very robustly.

Reply:
Cool, I haven't run into any issues with just splitting on . thus far but worth considering in the future if we run into any problems.